src/helpers/password.js

```diff
@@ -5,7 +5,7 @@ import bcrypt from 'bcrypt';
 *   @exports passwordHash
 * */
 
-const passwordHash = (password) => bcrypt.hashSync(password, Number(process.env.SALT));
+const passwordHash = password => bcrypt.hashSync(password, Number(process.env.SALT));
```

Comment:
Is it possible to use the async mode for Hashing? Also, can this be an Hash.js file which is a class Hash having to methods:

1. make(value) - returns the hashed value
2. compare(value, hash) - returns true or fals

`Why is async mode recommended over sync mode?
If you are using bcrypt on a simple script, using the sync mode is perfectly fine. However, if you are using bcrypt on a server, the async mode is recommended. This is because the hashing done by bcrypt is CPU intensive, so the sync version will block the event loop and prevent your application from servicing any other inbound requests or events. The async version uses a thread pool which does not block the main event loop.`